doc/latex/Setup.tex

```diff
@@ -708,308 +741,100 @@ \subsection{initialConcentrations.config} \label{subsec:initialconcentrations}
-\hyperref[sec:parameters]{Model Parameters}). The format of the
-corresponding output files -- \texttt{lossRates.output} and
-\texttt{productionRates.output} -- is designed to facilitate the
-analysis of production and destruction rates for selected species of
-interests (rather than processing the output files generated in
-\texttt{model/output/reactionRates/}):
-
-\newpage
-\begin{scriptsize}
-\begin{verbatim}
-     time        speciesNumber    speciesName    reactionNumber         rate        reaction
-
-3.600000E+003           8               OH              15         0.000000E+000    O1D=OH+OH
-3.600000E+003           8               OH              20         0.000000E+000    HO2+O3=OH
-3.600000E+003           9               HO2             16         0.000000E+000    OH+O3=HO2
-3.600000E+003           9               HO2             17         0.000000E+000    OH+H2=HO2
-
-7.200000E+003           8               OH              15         0.000000E+000    O1D=OH+OH
-7.200000E+003           8               OH              20         0.000000E+000    HO2+O3=OH
-7.200000E+003           9               HO2             16         0.000000E+000    OH+O3=HO2
-7.200000E+003           9               HO2             17         0.000000E+000    OH+H2=HO2
-\end{verbatim}
-\end{scriptsize}
+This file lists the chemical species for which detailed production and
+loss rates are required. In version 1.0 and earlier, these species
+were listed in two different files called
+\texttt{productionRatesOutput.config} and \texttt{lossRatesOutput.config}.
+The file has one column, with one species per line. The frequency of
+this output is controlled by the \textbf{rates output step size}
+parameter in \texttt{model.parameters} (Sect.~\ref{sec:model-parameters}).
+
+The corresponding output files -- called \texttt{productionRates.output}
+and \texttt{lossRates.output} -- are designed to facilitate the
+analysis of theproduction and destruction rates for selected species of
+interests (rather than processing all the files saved in the
+\texttt{model/output/reactionRates/} directory). For more information
+go to Sect.~\ref{sec:output}.
 
 \subsection{outputSpecies.config} \label{subsec:outputspecies}
 
-This file (called \texttt{concentrationOutput.config} in v1.0) lists
-the chemical species for which the model output is required. The
-current version of AtChem2 limits the number of species that can be
-output to 100, although the user can modify the Fortran code to
-increase this number. The file has one column, with one species per
-line.
+This file (called \texttt{concentrationOutput.config} in version 1.0
+and earlier) lists the chemical species for which the calculated
+concentration is required. The current version of AtChem2 limits the
+number of species that can be output to 100, although the user can
+modify the Fortran code to increase this number. The file has one
+column, with one species per line.  %% TODO: how to change the number
```

Comment:
@spco can you confirm that this is done by modifying line 375 of `outputFunctions.f90`?